.github/workflows/ci_workflow.yml

```diff
@@ -0,0 +1,19 @@
+name: Build and Run Test Application Workflow
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # (scheduled - once per day at midnight)
+  pull_request: # (for every push to Pull Request)
+    types: [opened, reopened, synchronize]
+  workflow_dispatch:  # (to start the workflow manually)
+
+jobs:
+  build_project:
+    name: Build Project
+    uses: ./.github/workflows/build_esp_app.yml
+# Disable this job if you don't use it for hardware testing
+  test_project:
+    if: ${{ false }}  # Disable this job as default
```

Comment:
Maybe we need to enable this by default, otherwise we won't notice if it gets broken.

(You can use a condition that namespace == espressif to keep this disabled for people forking the project)

Reply:
Done! Now the test workflow will be skipped if the repository owner is different from "Espressif".